services/transactions/transactions_test.go

```diff
@@ -0,0 +1,199 @@
+package transactions
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	transactionspb "github.com/openbank/gunk/gunk/v1/transactions"
+	"github.com/openbank/openbank/storage"
+)
+
+func TestHandler_GetTransaction(t *testing.T) {
+	type args struct {
+		ctx context.Context
+		req *transactionspb.GetTransactionRequest
+	}
+	tests := []struct {
+		name      string
+		args      args
+		mockCalls func(m *storage.MockTransactionStore)
+		want      *transactionspb.Transaction
+		wantErr   bool
+	}{
+		{
+			name: "get transaction ok",
```

Comment:
test names should never contain spaces, and it should follow the naming style of Go funcs. As it is now, this will probably print as:

`TestHandler_GetTransaction/get_transaction_ok`

I think a nicer name would be `TestHandler_GetTransaction/Success`, or something similar.